practice_6/task_4.html

```diff
@@ -0,0 +1,31 @@
+<html>
+  <head>
+    <title>Task 4</title>
+  </head>
+  <body>
+    <ul id="list">
+      <li>1</li>
+      <li>2</li>
+      <li>3</li>
+      <li>4</li>
+      <li>5</li>
+    </ul>
+    <script>
+        let listItems = document.getElementsByTagName('li');
+  
+        const result = [
+          listItems[0].textContent,
+          listItems[listItems.length - 1].textContent,
+          listItems[1].textContent,
+          listItems[3].textContent,
+          listItems[2].textContent,
+        ];
+  
+        alert('Результат виводу: ' + result.join(', '));
+    </script>
+  </body>
+</html>
+
+
```

Comment:
You should do this task in 2 ways according to task. I see only 1 way

Reply:
added second way